docs/sdk-and-tools/sdk-js/sdk-js-cookbook.md

```diff
@@ -260,6 +260,9 @@ const tx1 = factory.createESDTTransfer({
 ### Single NFT transfer
 
 ```
+// first arg - collection
```

Comment:
I think that the same comments should be added for other examples as well (for example: single SFT transfers)